### PR TITLE
Feature/create house

### PR DESCRIPTION
**Pull request by xingweitian**
Hi, I open this pr to solve #17 , I create two APIs: /house/create and /house/upload_photo.

I need to mention that the photo is stored in /media folder for now(It will be created automatically while the project is running), I think it's necessary to talk about whether storing in Mongodb is a better way.

What have I done?

- create /house/create

- create /house/upload_photo

In the end, I really want to say, Django SUCKS.

### CODE REVIEW

File: house/views.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+def create(request):
+    name = request.POST["house_name"]
+    place_id = request.POST["place_id"]
+    address = request.POST["house_address"]
+    city = request.POST["house_city"]
+    province = request.POST["house_province"]
+    postcode = request.POST["house_postcode"]
+    coordinate = request.POST["house_coordinate"]
+    if isinstance(coordinate, str):
+        coordinate = [float(i) for i in coordinate.lstrip("[").rstrip("]").split(",")]
```
Comment (zhangjiangqige) [inline house/views.py]: Using two parameters, longitude and latitude, would be clearer and easier than passing a string of list

File: house/views.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                "success": 1,
+                "info":{
+                    "house_id": str(house.pk),
+                    "geo_object_id": str(res)
+                }
+            })
+        else:
+            return JsonResponse({
+                "success": 0
+            }), 500
```
Comment (zhangjiangqige) [inline house/views.py]: I remember 500 should be inside of the bracket for Django?

Also we'd better print some log here.

File: db/dbutils.py [hunk 3]
```diff
@@ -9,4 +11,37 @@ def exists(model, **kwargs):
...
\ No newline at end of file
+    return True
+
+
+def db_connection():
+    return MongoClient(host=conf["database_uri"])
+
+
+DB = "birana"
+COLLECTION = "house_locations"
```
Comment (zhangjiangqige) [inline db/dbutils.py]: Maybe it's better to put them in utils.config.

**Review by zhangjiangqige — CHANGES_REQUESTED** [review]

File: config.json [hunk 4]
```diff
@@ -2,5 +2,6 @@
     "log_level": "ERROR",
     "database_name": "birana",
     "database_uri": "mongodb://teachertian:aGMpG4BW8hQU9np@ds331135.mlab.com:31135/birana",
-    "sentry_dsn": "https://3cf3191bbc8343f78d32edf5545d4226@sentry.io/1390859"
+    "sentry_dsn": "https://3cf3191bbc8343f78d32edf5545d4226@sentry.io/1390859",
+    "database_collection": "house_locations"
```
Comment (zhangjiangqige) [inline config.json]: I think we should use a more specific key name here, such as geometry_collection or house_location_collection etc.

**Review by zhangjiangqige — APPROVED** [review]

File: config.json (hunk 4 above)
Comment (xingweitian) [reply config.json]: Agree!

**Pull request merged**